draft-ietf-quic-http.md

```diff
@@ -1382,7 +1386,7 @@ HTTP_PUSH_ALREADY_IN_CACHE (0x04):
 : The server has attempted to push content which the client has cached.
 
 HTTP_REQUEST_CANCELLED (0x05):
-: The client no longer needs the requested data.
+: The sender is no longer processing the request or response.
```

Comment:
Not sure if you want to use "processing" here.  It might be read as implying something more (REQUEST_REJECTED).  Maybe:

```suggestion
: The request or its response is cancelled.
```